.github/workflows/main.yaml

```diff
@@ -0,0 +1,28 @@
+name: release workflow
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    name: release action
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: build binaries
+      uses: cedrickring/golang-action/go1.12@1.3.0
```

Comment:
why use this over using the official setup-go action?

https://github.com/actions/setup-go#usage

Reply:
Thanks. I'll take a look.